src/javascripts/components/forms/editBaggageForm.js

```diff
@@ -0,0 +1,27 @@
+const editBaggageForm = (baggageObject) => {
+  document.querySelector('#form-container').innerHTML = `
+    <div class="card bg-light mb-3" style="width: 40em;">
+    <div class="card-header"></div>
+    <div class="card-body">
+      <h5 class="card-title"><h3>Pan Am Claims Department</h3></h5>
+      <p class="card-text">Exclusive First & Clipper class check-in service.</p>https://github.com/nss-evening-cohort-14/pan-am-group-project/blob/update-baggage/src/javascripts/components/forms/editBaggageForm.js
```

Comment:
Is this supposed to be here? `https://github.com/nss-evening-cohort-14/pan-am-group-project/blob/update-baggage/src/javascripts/components/forms/editBaggageForm.js`

Reply:
Took this out. We are trying to figure out the bug on the airport before we push back up

Reply:
😎 Great!